cdap-docs/developers-manual/source/getting-started/standalone/virtual-machine.rst

```diff
@@ -20,26 +22,58 @@ The CDAP Standalone Virtual Machine is configured with the recommended settings
 
 - 4 GB of RAM
 - Ubuntu Desktop Linux
-- 10 GB of disk space
+- 40 GB of virtual disk space
 
```

Comment:
Well, it wasn't 10GB either. As we were giving the configuration, I changed it to the actual configuration I found. It's probably a sparse or an expanding drive, taking up as much space as it actually requires.